eslint.json

```diff
@@ -32,7 +32,7 @@
       "error",
       "unix"
     ],
-    "newline-before-return": "error",
+    "padding-line-between-statements": "error",
```

Comment:
All of these keys are alphabetically ordered.

Reply:
This rule does nothing if no configurations are provided. 

Read here. https://eslint.org/docs/rules/padding-line-between-statements

Reply:
We need this. Let me know if you suggest otherwise.

```json
[{ blankLine: "always", prev: ["const", "let", "var"], next: "*"}, { blankLine: "any", prev: ["const", "let", "var"], next: ["const", "let", "var"]}]
```